js/main.js

```diff
@@ -33,6 +33,36 @@ maps.forEach(map_ => {
               duration: 1.0
             });
         }, 1000)
+    
+
+        let tobacco_field_data = {
```

Comment:
Can we have this as `farm_field_data` seeing as there are other crops aside from tobacco

Reply:
that is done